Isolate Machine with Cybereason from Slack (#483)
rapid7/insightconnect-workflows: workflows/Isolate_Machine_with_Cybereason_from_Slack/help.md

# Description

This workflow allows a user to isolate a machine using Cybereason via a configurable Slack chatbot command.

Sample Slack Trigger Commands:

`@Rapid7 InsightConnect isolate <example_machine>`

# Key Features

* Isolate a machine with Cybereason from Slack

# Requirements

* [Slack](https://insightconnect.help.rapid7.com/docs/configure-slack-for-chatops)
* Cybereason account

# Documentation

## Setup

Import the workflow from the Rapid7 Extension Library and proceed through the Import Workflow wizard in InsightConnect. Import plugins, create or select connections, and rename the workflow as a part of the Import Workflow wizard as necessary.

> Once the workflow has been imported, you must configure the Workflow Parameters for the Slack chatops command as well as the channel the chatops bot should monitor. 

Reviewer: ```suggestion
Once the workflow has been imported, you must configure the Workflow Parameters for the Slack chatops command and the channel the chatops bot should monitor. 
```

Reviewer: sentence reads a little weird. I think this helps
